Userland/Libraries/LibWeb/CSS/Default.css

```diff
@@ -237,6 +237,15 @@ input[type=submit], input[type=button], input[type=reset], input[type=checkbox],
     cursor: unset;
 }
 
+input[type=button] {
+    padding: 2px 8px;
```

Comment:
I feel like this could use a little more vertical padding - 4px?

Reply:
I agree and adjusted this.